pkg/webhooks/webhooks.go

```diff
@@ -246,3 +255,24 @@ func (wh WebhookHandler) HandleMutatingNsDelete(ns *corev1.Namespace, dryRun boo
 	wh.logger.Info("HCO CR still exists, forbid namespace deletion")
 	return false, nil
 }
+
+func (wh WebhookHandler) validateCertConfig(hc *v1beta1.HyperConverged) error {
+	minimalDuration := metav1.Duration{Duration: 60 * time.Minute}
+	if hc.Spec.CertConfig.CA.Duration.Duration < minimalDuration.Duration ||
+		hc.Spec.CertConfig.CA.RenewBefore.Duration < minimalDuration.Duration ||
+		hc.Spec.CertConfig.Server.Duration.Duration < minimalDuration.Duration ||
+		hc.Spec.CertConfig.Server.RenewBefore.Duration < minimalDuration.Duration {
+		return errors.New("certConfig: values too small")
+	}
+
+	if hc.Spec.CertConfig.CA.Duration.Duration < hc.Spec.CertConfig.CA.RenewBefore.Duration ||
+		hc.Spec.CertConfig.Server.Duration.Duration < hc.Spec.CertConfig.Server.RenewBefore.Duration {
+		return errors.New("certConfig: duration < renewBefore")
```

Comment:
Personally, I prefer to use more verbose error messages. Can we use the words "earlier/later than" instead of > or < ?

Reply:
done